api/tests/mocha/server-utils.spec.js

```diff
@@ -1,18 +1,19 @@
-const sinon = require('sinon'),
-      chai = require('chai'),
-      environment = require('../../src/environment'),
-      serverUtils = require('../../src/server-utils'),
-      req = {
-        url: '',
-        get: () => {}
-      },
-      res = {
-        writeHead: () => {},
-        end: () => {},
-        json: () => {},
-        redirect: () => {},
-        status: () => {}
-      };
+const sinon = require('sinon');
```

Comment:
Sorry about the refactoring here. It's basically unchanged except for two new tests with the `#5809` suffix.